src/languages/en.js

```diff
@@ -825,6 +825,9 @@ export default {
         privateDescription: 'Only people invited to this room are able to find it',
         createRoom: 'Create Room',
         roomAlreadyExists: 'A room with this name already exists',
+        pleaseEnterRoomName: 'Please enter a room name',
+        pleaseSelectWorkspace: 'Please select a workspace',
+        pleaseSelectVisibility: 'Please select a visibility',
```

Comment:
I am not too sure about these messages. 
I think the best would be to show. `{field} is a required field`.

So `${this.props.translate(fieldNameKey)} ${this.props.translate('common.isRequiredField')` is better for all.

cc: @marcaaron

Reply:
For copy we tag in @Expensify/marketing to give us the exact copy. Translations I think we can ask for help in Slack.

Reply:
The first two look good. I'm not sure about `Please select a visibility`, but think there should be no error since it's a required field and only has two options `Restricted | Private`.

Reply:
Would the visibility be at the room-level? Maybe it could be something like "Please select a room type." Also, a couple questions:

1. Are these questions they'll get in this order? I'd imagine it would instead be more natural to have: workspace THEN room name THEN room type/visibility
2. Will we have descriptions of what "Restricted" and "Private" actually mean? Are there any other room types, or just these two?

Reply:
Thanks @jamesdeanexpensify! These are error messages that the user will see if they do not input correct values when they are creating a room.

To clarify, I meant to say "we don't need to worry about the visibility error" as there is a default selection made at all times (i.e. not possible to have a missing one).

Reply:
Ah, understood - so "Restricted" and "Private" are defined for users elsewhere in the product? Just making sure. And for the conversation at hand, are you just looking for signoff on the proposed copy:

1. Please enter a room name
2. Please select a workspace

If so, then (personal preference) I'd probably update "select" to "choose" so it's "Please choose a workspace" but other than that it looks fine to me! 

Last thing - I'm going to ask around to see if we've been writing these error handling messages with periods at the end since I haven't been involved in creating many lately.

Reply:
>you just looking for signoff on the proposed copy

Yep! Thanks!

>I'm going to ask around to see if we've been writing these error handling messages with periods at the end since I haven't been involved in creating many lately.

Sounds good. I'm not sure either. Probably there is inconsistency and we can create an issue to resolve it.

Reply:
Let's leave these without periods. Let me know if you need anything else, thanks!